meta-mender-qemu/scripts/manifest-qemu.xml

```diff
@@ -2,6 +2,7 @@
   <manifest>
 
   <remote fetch="git://git.yoctoproject.org" name="yocto"/>
+  <remote fetch="git://git.openembedded.org" name="meta-openembedded"/>
```

Comment:
Change to:

```
<remote fetch="git://git.openembedded.org"	     name="oe"/>
```

Reply:
right! and I did check `manifest-coral.xml` but I tried to be clever -- never works ;)